crates/trippy-tui/src/config.rs

```diff
@@ -811,11 +811,16 @@ const fn cfg_layer_bool_flag(fst: bool, snd: Option<bool>, default: bool) -> boo
 }
 
 /// Check for deprecated fields.
-fn validate_deprecated(cfg_file_tui: &ConfigTui) -> anyhow::Result<()> {
+fn validate_deprecated(
+    cfg_file_tui: &ConfigTui,
+    cfg_file_tui_bindings: &ConfigBindings,
+) -> anyhow::Result<()> {
     if cfg_file_tui.deprecated_tui_max_samples.is_some() {
         Err(anyhow!("tui-max-samples in [tui] section is deprecated, use max-samples in [strategy] section instead"))
     } else if cfg_file_tui.deprecated_tui_max_flows.is_some() {
         Err(anyhow!("tui-max-flows in [tui] section is deprecated, use max-flows in [strategy] section instead"))
+    } else if cfg_file_tui_bindings.deprecated_toggle_privacy.is_some() {
```

Comment:
I'm wondering if this would be easier to read as a match statement?

Reply:
I don't think so:

```rust
    match (
        cfg_file_tui.deprecated_tui_max_samples.is_some(),
        cfg_file_tui.deprecated_tui_max_flows.is_some(),
        cfg_file_tui_bindings.deprecated_toggle_privacy.is_some(),
    ) {
        (true, _, _) => Err(anyhow!("tui-max-samples in [tui] section is deprecated, use max-samples in [strategy] section instead")),
        (_, true, _) => Err(anyhow!("tui-max-flows in [tui] section is deprecated, use max-flows in [strategy] section instead")),
        (_, _, true) => Err(anyhow!("toggle-privacy in [bindings] section is deprecated, use expand-privacy and contract-privacy instead")),
        _ => Ok(()),
    }
```